src/MentionsInput.js

```diff
@@ -500,7 +506,7 @@ class MentionsInput extends React.Component {
         return;
       }
 
-      const regex = _getTriggerRegex(child.props.trigger);
+      const regex = _getTriggerRegex(child.props.trigger, that.props);
```

Comment:
I know `that` was already there before. But since we're all good boy scouts could you please fix that so that the code uses an arrow function and `this` is automatically mapped to the right value?